asp_interactive_configuration/ooasp/run.py

```diff
@@ -155,7 +184,7 @@ def get_parser() -> ArgumentParser:
 
     end = time.time()
 
-    out_name = f"benchmarks/latest/rD{args.racksD}-rS{args.racksS}-e{args.element}"
+    out_name = f"benchmarks/latest/f-{args.frame}r-{args.rack}-rD{args.racksD}-rS{args.racksS}-e{args.element}-eA{args.elementA}-eB{args.elementB}-eC{args.elementC}-eD{args.elementD}-m{args.module}-mI{args.moduleI}-mII{args.moduleII}-mIII{args.moduleIII}-mIV{args.moduleIV}-mV{args.moduleV}" #this will eventually be too long, so this naming conv. might have to be changed
```

Comment:
I have created a function for file name generation, that filters out all arguments with the value 0 and builds the new filename in an automated cycle in order to avoid inconsistencies.